src/main/webapp/app/quiz/participate/quiz.component.html

```diff
@@ -75,7 +75,7 @@ <h2 [ngSwitch]="mode">
             <div *ngIf="mode === 'solution'">
                 <span jhiTranslate="arTeMiSApp.quizExercise.solution"></span>
             </div>
-            <button *ngIf="!submission.submitted && !showingResult" [disabled]="isSubmitting || waitingForQuizStart || remainingTimeSeconds < 0" class="btn btn-success btn-lg" jhiTranslate="entity.action.submit" (click)="onSubmit()"></button>
+            <button *ngIf="!submission.submitted && !showingResult" [disabled]=" waitingForQuizStart || remainingTimeSeconds < 0" class="btn btn-success btn-lg" jhiTranslate="entity.action.submit" (click)="onSubmit()"></button>
```

Comment:
Why are you removing this check? **This is wrong!** While the user is submitting, the button should be disabled. You have to make sure though that isSubmitting is maintained correctly in the onSubmit() function below. You only set this variable to true, if the actual submit has happened (i.e. if the user has confirmed) and not before

Reply:
isSubmitting is back in place - and the button is disabled when someone is submitting - also it reappears when cancel is clicked